json-reporter.js

```diff
@@ -0,0 +1,30 @@
+class MyCustomReporter {
+  constructor(globalConfig, options) {
+    this._globalConfig = globalConfig;
+    this._options = options;
+  }
+
+  onRunComplete(contexts, results) {
+    results.testResults[0].testResults.forEach((item) => {
+      if (item.status !== "failed") {
+        return;
+      }
+      const newLine = "%0A";
+      const name = results.testResults[0].testFilePath;
+
+      if (!item.location) {
+        throw new Error(
+          "You need to invoked jest with argument --testLocationInResults"
```

Comment:
By default the line number is not part of the output. Since https://github.com/facebook/jest/pull/4782 there is an option `--testLocationInResults` to include those values as well. I wonder if there is another way to opt-in into this behaviour via options

Reply:
This might be something we can let reporters decide. E.g. read it a static property on the reporter or something.

Thinking about it, we might be able to make the behavior default... Not sure

Reply:
FYI @SimenB Now that I lookup the line number from the stack trace, this isn't needed anymore

Reply:
👍